ci-operator/step-registry/upi/install/powervs/sno/upi-install-powervs-sno-commands.sh

```diff
@@ -91,8 +91,15 @@ SSH_OPTIONS=(-o 'PreferredAuthentications=publickey' -o 'StrictHostKeyChecking=n
 # https://codeload.github.com/ppc64le-cloud/ocp-sno-hacks/zip/refs/heads/main contains the scripts to setup the SNO cluster
 ssh "${SSH_OPTIONS[@]}" root@${BASTION} "mkdir ${BASTION_CI_SCRIPTS_DIR} && cd ${BASTION_CI_SCRIPTS_DIR} && curl https://codeload.github.com/ppc64le-cloud/ocp-ci-hacks/zip/refs/heads/main -o ocp-ci-hacks.zip && unzip ocp-ci-hacks.zip && mv ocp-ci-hacks-main/sno/* ${BASTION_CI_SCRIPTS_DIR} && rm -rf ocp-ci-hacks.zip ocp-ci-hacks-main"
```

Comment:
This makes this PR clean, but it hides the logic of what is going on.
One weakness of this approach to AI testing is that it won't test the desired nightly payload. Rather, it will only be able to target release payloads from the public console. This is OK, from the perspective of checking weather or note a released version is broken, but it reduces the value of the job.